src/_posts/2021-01-19-bootstrap-icons-4.6.0.md

```diff
@@ -0,0 +1,135 @@
+- [#32462](https://github.com/twbs/bootstrap/pull/32462): Replace Lorem Ipsum placeholder text with more representative (or at least english language) text
+- [#32634](https://github.com/twbs/bootstrap/pull/32634): Remove incorrect mention of dropdowns for dynamic tab behavior
+- [#32639](https://github.com/twbs/bootstrap/pull/32639): v4: Add an actual `data-touch="false"` example in the carousel docs
+- [#32728](https://github.com/twbs/bootstrap/pull/32728): add v5.0 in versions
+- [#32761](https://github.com/twbs/bootstrap/pull/32761): Mention stretched-link constraints with table elements
+- [#32789](https://github.com/twbs/bootstrap/pull/32789): Remove `role="button"` from CTA links in carousel example
+- [#32791](https://github.com/twbs/bootstrap/pull/32791): Docs v4: Sass implementation and rounding precision
+- [#32809](https://github.com/twbs/bootstrap/pull/32809):
+  - Clarify Sass import and customize docs for how to modify variable defaults
+  - Add an npm starter project callout to a few pages
+- [#32827](https://github.com/twbs/bootstrap/pull/32827): Add a live toast example to the docs
+- docs(dropdowns): clarify where is `.show` applied
+- Require `.has-validation` for input groups with validation
+- Fix mobile menu jump & double border
+- Remove double spaces from homepage SVGs
+- browserconfig.xml: switch to relative image path
+- Tweak the wording for collapse to indicate button is preferred/more semantic; backport of [#32632](https://github.com/twbs/bootstrap/pull/32632)
+- Clarify the `$enable-shadows` option in our docs; backport of [#32685](https://github.com/twbs/bootstrap/pull/32685)
+
+### Examples
+
+- [#31979](https://github.com/twbs/bootstrap/pull/31979): v4 Examples/Floating-labels: fix bad behavior with autofill
+- [#32198](https://github.com/twbs/bootstrap/pull/32198): examples: add the version number in `title`
+
+### Misc
+
+- [#29753](https://github.com/twbs/bootstrap/pull/29753): Improve build/generate-sri.js regex
+- [#32003](https://github.com/twbs/bootstrap/pull/32003): CI: switch to Node.js 14.
+- [#32008](https://github.com/twbs/bootstrap/pull/32008): Update Edge's Rendering Engine on CONTRIBUTING.md
+- [#32486](https://github.com/twbs/bootstrap/pull/32486): BrowserStack: test on macOS Catalina instead of High Sierra
+- [#32756](https://github.com/twbs/bootstrap/pull/32756): Stylelint: disallow some property values
+- **Fix for npm 7.x** package.json: move `version_short` variable under the `config` object; backport of [#32737](https://github.com/twbs/bootstrap/pull/32737)
+- Update build-examples script so that the resulting examples zip file includes only the needed files
+- Various CI tweaks
+- Updated devDependencies
+
+[Head to the v4.6 docs](https://getbootstrap.com/docs/4.6/) to see the latest in action. The full release has been published to npm and will soon appear on jsDelivr and Rubygems.
```

Comment:
Might need some rewording here since jsDelivr is automatically updated, so only Rubygems can be behind